tests/eval-test.sh

```diff
@@ -1,15 +1,11 @@
 #!/bin/sh
+set -efu
+cd "$(dirname "$0")/.." || exit 1
 
-set -e
-
-cd "$(dirname "$0")/.."
+echo "### Evaluating all profiles ###"
+echo
 
+# shellcheck disable=SC2044
```

Comment:
Not sure why you ignore this check: https://github.com/koalaman/shellcheck/wiki/SC2044

In this case, it could be rewritten as `find . -name default.nix -exec tests/eval-one.sh {}`. But again, I think we shouldn't accept files with weird names into this repo, and current shell script makes sure that we won't.

Reply:
There is no solution that doesn't depend on word splitting, without depending on bash 4.  `find -exec` doesn't early-return nor propagate the exit status. I don't think that profiles should contain special characters in any case.